### PR TITLE
Feature/jl leader boards

### PR DESCRIPTION
**Pull request by jorelllinsangan**

### CODE REVIEW

File: code/server/routes/index.js [hunk 1]
```diff
@@ -28,6 +28,36 @@ router.get("/matches", function(req, res) {
 	});
 });
 
+router.get("/leaderBoard", function(req,res) {
```
Comment (KeaneTK) [inline code/server/routes/index.js]: change "/leaderBoard" to "/leaders".  Didn't work on my computer until I did so.

Comment (jorelllinsangan) [inline code/server/routes/index.js]: That's really weird. I'm curious, what was happening before you changed it?

Comment (KeaneTK) [inline code/server/routes/index.js]: Just a white page with "Cannot GET /leaders"

Comment (sbairos) [inline code/server/routes/index.js]: Can confirm, yeah. Do what Keane says

**Pull request merged**